feedback/feedbackController.js

```diff
@@ -0,0 +1,34 @@
+import { createGitHubIssue } from './githubService.js'
+import { respondWithError } from "../utilities/shared.mjs"
+
+export async function submitFeedback(req, res) {
+  const user = req.user ? `${req.user.profile.displayName} (${req.user._id})` : 'Anonymous'
+  const { page, feedback } = req.body
+
+  if (!feedback) {
+    return res.status(204).send()
```

Comment:
Ah cause this send() won't stop the rest of the code from executing.  Probably no real difference in returning this vs using `return` on the next line.

Reply:
I use return on the next line for next() calls, but for sending the response, it feels more normal this way. Happy to move it.